src/core_atmosphere/physics/mpas_atmphys_manager.F

```diff
@@ -548,11 +548,13 @@ subroutine physics_run_init(configs,mesh,state,clock,stream_manager)
  endif
 
 !set alarm for updating the background surface albedo and the greeness fraction:
- call mpas_set_timeInterval(alarmTimeStep,timeString=config_greeness_update,ierr=ierr)
- alarmStartTime = startTime
- call mpas_add_clock_alarm(clock,greenAlarmID,alarmStartTime,alarmTimeStep,ierr=ierr)
+ if(trim(config_greeness_update) /= "none") then
```

Comment:
@barlage If config_greeness_update is set to 'none' the model crashes. This is a problem in MPAS. Therefore, to run the model config_greeness_update should always be not 'none', and this additional IF that you propose is not needed.

Reply:
@tanyasmirnova the model will not crash with change; as noted I have tested it and it works. The model crashes because of the call inside this if block.

Reply:
@barlage Did you try to set config_greeness_update = none?

Reply:
Yes, my test directory with the namelist is here:
jet: /lfs5/BMC/wrfruc/Michael.Barlage/mpas/vegfra_test/run_config_test

Reply:
I compile with GNU and my executable doesn't work with config_greeness_update = none. What compiler do you use?

Reply:
also, if we set config_greeness_update = none, albedo will not be updated, right? My version of the code separated aldbedo from the greenness updating.

Reply:
I'm using ifort. Are you using `none` or `"none"`? It needs to be a string.

Do we need to update albedo in these short simulations? To me, separating albedo from greeness is a different issue. The way the code is currently constructed, there is only one alarm that updates both albedo and greeness. If the goal is to update them at separate cadences, then there should be a separate alarm for each.

Reply:
I used "none". 
I hope you can approve my PR,  because it does a minimal change to the MPAS code, easy to merge. I tested it for several cases, and GVF is passed along fine, plus it is already used in real-time MPAS. 
In future we could continue looking for the best solution on how to treat surface updating. I think GVF should be separate from albedo.

Reply:
before Anders and I approve/reject, can we at least square the ifort vs gnu difference?

Reply:
I compile with GNU as I did from the very beginning.
When I add to namelist.atmosphere:    config_greeness_update = "none", the model immediately crashes:

Due to MODULEPATH changes, the following have been reloaded:
  1) netcdf/4.7.0

The following have been reloaded with a version change:
  1) intel/2022.1.2 => intel/2023.2.0

forrtl: severe (59): list-directed I/O syntax error, unit -5, file Internal List-Directed Read
Image              PC                Routine            Line        Source
atmosphere_model   0000000000CE55ED  for_read_int_lis_     Unknown  Unknown
atmosphere_model   0000000000CE4B7C  for_read_int_lis      Unknown  Unknown
atmosphere_model   0000000000B70096  Unknown               Unknown  Unknown
atmosphere_model   0000000000631DBC  Unknown               Unknown  Unknown
atmosphere_model   00000000004E2C18  Unknown               Unknown  Unknown
atmosphere_model   00000000004E4139  Unknown               Unknown  Unknown
atmosphere_model   000000000041AE64  Unknown               Unknown  Unknown
atmosphere_model   0000000000417C39  Unknown               Unknown  Unknown
atmosphere_model   0000000000417BCD  Unknown               Unknown  Unknown
libc-2.28.so       0000145A62412D85  __libc_start_main     Unknown  Unknown
atmosphere_model   0000000000417AEE  Unknown               Unknown  Unknown
forrtl: severe (59): list-directed I/O syntax error, unit -5, file Internal List-Directed Read

The model can't read the namelist.
Ming explained to me that config_greeness_update expects the date, and that is why the model crashes.

Because of this crash, I added one more namelist option config_gvf_update, logical, and set it to false so that the GVF from RAP is not overwritten by climatology in MPAS. I pass it into mpas_atmphys_update_surface.F subroutine and if it is false GVF is not updated. A very minor change which works, plus it "de-couples" GVF from albedo, which is good, because we can treat GVF differently.
 
It appears that config_greeness_update = "none" works with intel, but I think that GNU is a more strict compiler. And anyway we have to treat GVF separately from albedo, therefore I like my solution better.

Reply:
I'll leave it up to the code managers to decide. My intention was not that this PR be merged, but was using this venue to demonstrate that there is a cleaner solution to the original problem. And I feel it is a solution that we will not have to manually deal with in future merges since it is only two lines and doesn't involve new namelist options. Also, see my comment below about contributing this back to NCAR.

I have compiled this two line change using GNU on Hercules and successfully run the test case. You can see the test environment here:
hercules: /work/noaa/stmp/mbarlage/mpas/vegfra_test
My Intel test environment on Jet is here:
jet:/lfs5/BMC/wrfruc/Michael.Barlage/mpas/vegfra_test

I see no reason why an if statement shouldn't work between compilers. 

The unchanged code (default test) looks like this (note that if I hadn't modified the IC file all these numbers would be the same):
```
 vegfra =
  18.36361,
  16.18756,
  16.18756,
  16.18756,
  16.18756,

```
The changed code with no namelist change (so 24 hour update) looks like this:
```
 vegfra =
  18.36361,
  16.18756,
  16.18756,
  16.18756,
  16.18756,

```
The changed code with config='none' test looks like this:
```
 vegfra =
  18.36361,
  18.36361,
  18.36361,
  18.36361,
  18.36361,

```
So I've tested this with both Intel and GNU and it works as expected.

As I mentioned earlier, the if block with 'none' is just following the same approach that is done with other namelist options. There are 7 options where you can use `‘DD_HH:MM:SS’ or ‘none’` as input. The others prevent 'none' from going into the alarm calculator because that is where the model will crash. I consider this a bug and am planning to create an issue and PR back to NCAR. If we want to deal with this later in the GSL fork, that is fine with me.

Reply:
I am not making it up that the model crash with config_greeness_update = "none" on jet with GNU compiler. You can see my test in /mnt/lfs5/BMC/wrfruc/smirnova/2024081618/fcst/ctl.
Again, my change is also de-coupling GVF from albedo.

Reply:
Top of the gsl/develop

Reply:
The namelist in this directory has `config_gvf_update`. Is this what is making the namelist read crash? That is not in gsl/develop

Reply:
I added config_gvf_update in my PR. It does not make the model crash when you use my PR.

Reply:
> I'll leave it up to the code managers to decide. My intention was not that this PR be merged, but was using this venue to demonstrate that there is a cleaner solution to the original problem. And I feel it is a solution that we will not have to manually deal with in future merges since it is only two lines and doesn't involve new namelist options. Also, see my comment below about contributing this back to NCAR.
> 
> I have compiled this two line change using GNU on Hercules and successfully run the test case. You can see the test environment here: hercules: /work/noaa/stmp/mbarlage/mpas/vegfra_test My Intel test environment on Jet is here: jet:/lfs5/BMC/wrfruc/Michael.Barlage/mpas/vegfra_test
> 
> I see no reason why an if statement shouldn't work between compilers.
> 
> The unchanged code (default test) looks like this (note that if I hadn't modified the IC file all these numbers would be the same):
> 
> ```
>  vegfra =
>   18.36361,
>   16.18756,
>   16.18756,
>   16.18756,
>   16.18756,
> ```
> 
> The changed code with no namelist change (so 24 hour update) looks like this:
> 
> ```
>  vegfra =
>   18.36361,
>   16.18756,
>   16.18756,
>   16.18756,
>   16.18756,
> ```
> 
> The changed code with config='none' test looks like this:
> 
> ```
>  vegfra =
>   18.36361,
>   18.36361,
>   18.36361,
>   18.36361,
>   18.36361,
> ```
> 
> So I've tested this with both Intel and GNU and it works as expected.
> 
> As I mentioned earlier, the if block with 'none' is just following the same approach that is done with other namelist options. There are 7 options where you can use `‘DD_HH:MM:SS’ or ‘none’` as input. The others prevent 'none' from going into the alarm calculator because that is where the model will crash. I consider this a bug and am planning to create an issue and PR back to NCAR. If we want to deal with this later in the GSL fork, that is fine with me.

@barlage Mike, could you please also print albedo with your change in the namelist?